#24 accept rqdata during init (#25)
hCaptcha/HCaptcha-ios-sdk: README.md

You can then enable it in your code:

``` swift
let hcaptcha = try? HCaptcha(
    ...

>     endpointApi: URL("https://custom.api.endpoint")!,

Reviewer: ```suggestion
    jsSrc: URL("https://js.hcaptcha.com/1/api.js")!,
```

.. but this should be optional, the default is correct for anyone not using first-party hosting.

>     endpoint: URL("https://custom.endpoint")!,

Reviewer: Same comment as above.

Author: I have removed `endpointApi`

Also according the text above it's required for enterprise clients

```
If you are an Enterprise user with first-party hosting access, you will need to set your own endpoint.

You can then enable it in your code:
```

Is that true?

Reviewer: It is only true *if that particular feature is used* (which is optional).

    ...
)

...
```

#### More params for Enterprise

More params like:

>  - `sentry`
>  - `rqdata`
>  - `reportapi`
>  - `assethost`
>  - `imghost`

Reviewer: ```suggestion
 - `sentry` (bool)
 - `rqdata` (string)
 - `reportapi` (string)
 - `assethost` (string)
 - `imghost` (string)
```

Can be passed via `HCaptcha(...)`

> Checout documentation in docs and https://docs.hcaptcha.com/api for more details

Author: @e271828- This link looks relevant, but I would like to doublecheck if it make sense here

> Checout documentation in docs and https://docs.hcaptcha.com/api for more details

Reviewer: ```suggestion
Please see the [hCaptcha Enterprise documentation](https://docs.hcaptcha.com/enterprise/) for more details.
```
